### PR TITLE
FALCON-2268 Extension list api does not follow default ASC order

### PR DESCRIPTION
**Pull request by PracheerAgarwal**

### CODE REVIEW

**Comment by PraveenAdlakha** [comment]
Please paste the exact commands you used to dev test this one.

File: prism/src/main/java/org/apache/falcon/resource/AbstractExtensionManager.java [hunk 1]
```diff
@@ -119,7 +117,7 @@ public int compare(ExtensionJobsBean o1, ExtensionJobsBean o2) {
             }
         };
 
-        Map<String, String> jobAndExtensionNames = new HashMap<>();
+        TreeMap<String, String> jobAndExtensionNames = new TreeMap<>();
```
Comment (pallavi-rao) [inline prism/src/main/java/org/apache/falcon/resource/AbstractExtensionManager.java]: Won't this cause the jobs be returned in ascending order even when user says desc?

**Comment by PracheerAgarwal** [comment]
falcon extension -list -extensionName extension-example
Hadoop is installed, adding hadoop classpath to falcon classpath
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/usr/local/falcon/falcon-distributed-0.11-SNAPSHOT/client/lib/slf4j-log4j12-1.7.5.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/usr/hdp/2.2.4.2-2/hadoop/lib/slf4j-log4j12-1.7.5.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
3
sampleAsc    extension-example
sampleAsc1    extension-example
sample    extension-example

**Comment by pallavi-rao** [comment]
This has been merged. @PracheerAgarwal, please close this PR.

**Pull request closed without merging**